arm-software/embedded/scripts/build_llvmlibc_toolchain.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright (c) 2025, Arm Limited and affiliates.
+# Part of the Arm Toolchain project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Build Arm Toolchain for Embedded with llvmlibc as the default C library
+
+# The script creates a build of the toolchain in the 'build_llvmlibc_toolchain'
+# directory, inside the repository tree.
+
+set -ex
+
+export CC=clang
+export CXX=clang++
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+REPO_ROOT=$( git -C ${SCRIPT_DIR} rev-parse --show-toplevel )
```

Comment:
Shell quoting nits: `${SCRIPT_DIR}` in this line should be double-quoted. So should `${BUILD_DIR}` in the `mkdir` and `cd` commands below.

(But it's OK not to quote these `$(...)` constructs on the RHS of an assignment, or the `${REPO_ROOT}` in the assignment to `BUILD_DIR`, because in that context, the shell decides what's the RHS of a variable assignment _before_ expanding those things into stuff that might contain spaces.)